ompi/mpiext/example/configure.m4

```diff
@@ -32,3 +32,11 @@ AC_DEFUN([OMPI_MPIEXT_example_CONFIG],[
 
 # only need to set this if the component needs init/finalize hooks
 AC_DEFUN([OMPI_MPIEXT_example_NEED_INIT], [1])
+
+# By default, mpiext_example_mpifh.h is included in the source file
+# of mpi_ext module. To disable it, define this macro.
+#AC_DEFUN([OMPI_MPIEXT_example_NO_MPIFH_IN_USEMP], [1])
```

Comment:
Can we reverse the names/logic?  Positive logic is always easier to read than negative logic.

E.g., `OMPI_MPIEXT_example_INCLUDE_MPIFH_IN_USEMPI` (vs. vs `NO` version).

Also, can we *always* define these values (i.e., to 0 or 1), vs. define them to 1 or undefine them?  This allows using `#if OMPI_MPIEXT_example_INCLUDE_MPFIH_IN_USEMPI` in source code, and you'll get a preprocessor error if you have made a typo in that name (which you won't get if you `#ifdef A_MACRO_NAME_WITH_A_TYPO_THAT_DOES_NOT_EXIST`).

Reply:
@jsquyres I reversed the name/logic. Please review again.

This macro is an Autoconf macro which is used only in `ompi_ext.m4`, not a C preprocessor macro. We never see the preprocessor error.